3rdParty/fuerte/src/HttpConnection.h

```diff
@@ -107,6 +125,17 @@ class HttpConnection final : public fuerte::GeneralConnection<ST> {
   http_parser_settings _parserSettings;
 
   std::atomic<bool> _active; /// is loop active
+  bool _reading;    // set between starting an asyncRead operation and executing
+                    // the completion handler
+  bool _writing;    // set between starting an asyncWrite operation and executing
+                    // the completion handler
+  // both are used in the timeout handlers to decide if the timeout still
+  // has to have an effect or if it is merely still on the iocontext and is now
+  // obsolete.
+  std::atomic<int>
+      _leased;  // set to 1 by the `lease` method, prevents idle alarm if
```

Comment:
The value of `_leased` is nowhere initialized, as far as I can see. It should be initialized in the ctor of HttpConnection.

Reply:
Right, done.